Dokan/Dokan/Scenes/ProductDetails/SliderView/SliderViewModel.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  SliderViewModel.swift
+//  Dokan
+//
+//  Created by heba isaa on 10/08/2022.
+//
+
+import Foundation
+import UIKit
+
+struct sliderViewModel {
```

Comment:
I would prefer adding this view model as a nested type in `SliderCollectionViewCell`

Example: 
```swift
extension SliderCollectionViewCell { 
    struct ViewModel { 
        let productImage: UIImage
        let noOfPictures: Int
        let noOfCurrentPicture: Int
    }
}
```